Dockerfile

```diff
@@ -1,6 +1,11 @@
 FROM node:latest
 
-ADD . /app
+RUN mkdir -p /app
 WORKDIR /app
 
+COPY package.json /app/
 RUN npm install
+
+COPY . /app
```

Comment:
The `.dockerignore` file includes `node_modules`, so it won't be copied

> Before the docker CLI sends the context to the docker daemon, it looks for a file named .dockerignore in the root directory of the context. If this file exists, the CLI modifies the context to exclude files and directories that match patterns in it. This helps to avoid unnecessarily sending large or sensitive files and directories to the daemon and potentially adding them to images using ADD or COPY.